src/node/chainstate.cpp

```diff
@@ -129,8 +129,8 @@ std::optional<ChainstateLoadVerifyError> VerifyLoadedChainstate(ChainstateManage
                                                                 bool fReset,
                                                                 bool fReindexChainState,
                                                                 const Consensus::Params& consensus_params,
-                                                                unsigned int check_blocks,
-                                                                unsigned int check_level,
+                                                                int check_blocks,
+                                                                int check_level,
```

Comment:
should `assert(check_blocks >= 0)` and `assert(check_level >= 0)` be added?

Reply:
No, this will crash the test